readthedocs/telemetry/collectors.py

```diff
@@ -89,8 +90,56 @@ def collect(self):
                 "all": all_apt_packages,
             },
         }
+        data["doctool"] = {
+            "name": self._get_doctool_name(),
+            "extensions": self._get_doctool_extensions(),
+            "theme": self._get_doctool_theme(),
+        }
         return data
 
+    def _get_doctool_name(self):
+        if self.config.sphinx:
+            return "sphinx"
+
+        if self.config.mkdocs:
+            return "mkdocs"
+
+        return "generic"
```

Comment:
This should probably make use of `version.documentation_type`, there we have the final doctype and doesn't matter what config file version you are using. 

https://github.com/readthedocs/readthedocs.org/blob/bb9dd8bf9bebbe5da0c747bee1e0a36d70335c4e/readthedocs/projects/tasks/builds.py#L531

and you could use these helpers

https://github.com/readthedocs/readthedocs.org/blob/bb9dd8bf9bebbe5da0c747bee1e0a36d70335c4e/readthedocs/builds/models.py#L380-L386

Reply:
Cool! I updated the code to use `is_sphinx_type` and `is_mkdocs_type` 👍🏼